packages/api/src/auth/auth.controller.spec.ts

```diff
@@ -49,449 +30,78 @@ describe("AuthController", () => {
-      expect(issuedAtValues.size).toBe(3);
-    });
-
-    it("returns a message with scheme http in development", async () => {
-      const newConfigServiceValues = {
-        ...configServiceValues,
-        NODE_ENV: "development",
-      };
-      configServiceMock = mock<ConfigService>({
-        get: jest.fn().mockImplementation((key: string) => newConfigServiceValues[key]),
-      });
-      const controller = new AuthController(configServiceMock);
-      const message = controller.getMessage(req, { address });
-      const siweMessage = new SiweMessage(message);
-      expect(siweMessage.scheme).toBe("http");
-    });
-
-    it("sets the new message in the cookie", async () => {
-      const message = controller.getMessage(req, { address });
-      expect(req.session.siwe).toEqual(new SiweMessage(message));
-    });
-
-    it("replaces old message with a new message each time", async () => {
-      const message1 = controller.getMessage(req, { address });
-      const message2 = controller.getMessage(req, { address });
-      expect(message1).not.toEqual(message2);
-      expect(req.session.siwe).toEqual(new SiweMessage(message2));
-    });
-
-    it("clears session when message is requested", async () => {
-      const nonce = "8r2cXq20yD3l5bomR";
-      const privateKey = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80";
-
-      const { siwe } = await calculateSiwe({
-        nonce,
-        privateKey,
-        domain: "blockexplorer.com",
-        scheme: "http",
-        chainId,
-      });
-      req.session = { siwe, verified: true };
-      const message = controller.getMessage(req, { address });
-      expect(req.session.siwe).toEqual(new SiweMessage(message));
-      expect(req.session.verified).toBe(undefined);
-    });
-
-    it("throws when address is invalid", async () => {
-      expect(() => controller.getMessage(req, { address: "0x123" })).toThrow(BadRequestException);
-    });
-
-    it("throws when address has invalid checksum", async () => {
-      expect(() => controller.getMessage(req, { address: "0xF39Fd6e51aad88F6F4ce6aB8827279cffFb92266" })).toThrow(
-        BadRequestException
-      );
-    });
-  });
-
-  describe("verifySignature", () => {
+  describe("login", () => {
     let body: VerifySignatureDto;
     let fetchSpy: jest.SpyInstance;
 
     beforeEach(() => {
-      body = new VerifySignatureDto();
+      body = { token: mockToken };
       fetchSpy = jest.spyOn(global, "fetch");
     });
 
     afterEach(() => {
       fetchSpy.mockRestore();
     });
 
-    it("throws when there is no message", async () => {
-      await expect(() => controller.verifySignature(body, req)).rejects.toThrow(BadRequestException);
-    });
-
-    it("validates for http in development", async () => {
-      // Mock the whitelist check to succeed
-      fetchSpy.mockResolvedValue({
-        ok: true,
+    it("should login successfully with valid token", async () => {
```

Comment:
Let's stick to the existing wording style for test cases:
`should login ...` -> `logins`
`should throw ...` -> `throws`

This way, the test cases read like a specification. Same in other places.